dds_cli/account_manager.py

```diff
@@ -272,21 +292,32 @@ def fix_project_access(self, email, project):
         except simplejson.JSONDecodeError as err:
             raise dds_cli.exceptions.ApiResponseError(message=str(err))
 
+        errors = response_json.get("errors")
+        error_messages = dds_cli.utils.parse_project_errors(errors=errors)
+
         if not response.ok:
             message = f"Failed updating user '{email}' project access"
             if response.status_code == http.HTTPStatus.INTERNAL_SERVER_ERROR:
                 raise dds_cli.exceptions.ApiResponseError(message=f"{message}: {response.reason}")
 
-            raise dds_cli.exceptions.DDSCLIException(
-                message=f"{message}: {response_json.get('message', 'Unexpected error!')}"
-            )
+            message += ": " + response_json.get("message", "Unexpected error!")
+            show_warning = True
+            if error_messages:
+                message += f"\n{error_messages}"
+                show_warning = False
 
-        LOG.info(
-            response_json.get(
+            raise dds_cli.exceptions.DDSCLIException(message=message, show_emojis=show_warning)
+
+        if errors:
```

Comment:
Same as above: `error_messages `instead of `errors` could be used or am I missing something?

Reply:
True. I'll have a look again.

Reply:
Fixed.